crates/sui-config/src/node.rs

```diff
@@ -207,6 +207,12 @@ pub struct NodeConfig {
     /// If unspecified, this will default to `128`.
     #[serde(default = "default_local_execution_time_channel_capacity")]
     pub local_execution_time_channel_capacity: usize,
+
+    /// Size of the LRU cache used for storing local execution time observations.
+    ///
+    /// If unspecified, this will default to `10000`.
+    #[serde(default = "default_local_execution_time_lru_cache_size")]
+    pub local_execution_time_lru_cache_size: usize,
```

Comment:
I would remove "LRU" from the names here, the type of cache isn't so important and who knows if we want to change it later